benches/filter.rs

```diff
@@ -1,14 +1,15 @@
 #![feature(test)]
 extern crate grcov;
+extern crate rustc_hash;
 extern crate test;
 
 use grcov::{CovResult, Function};
-use std::collections::HashMap;
+use rustc_hash::FxHashMap;
 use test::{black_box, Bencher};
 
 #[bench]
 fn bench_filter_covered(b: &mut Bencher) {
-    let mut functions: HashMap<String, Function> = HashMap::new();
+    let mut functions:FxHashMap<String, Function> = FxHashMap::default();
```

Comment:
Nit: keep the space between the variable name and the type (`functions: FxHashMap...`)

Reply:
Oops! I thought code would catch something like this